src/sql.rs

```diff
@@ -797,6 +797,19 @@ async fn prune_tombstones(sql: &Sql) -> Result<()> {
     Ok(())
 }
 
+/// Helper function to return comma-separated sequence of `?` chars.
+///
+/// Use this together with [`rusqlite::ParamsFromIter`] to use dynamically generated
+/// parameter lists.
+pub fn repeat_vars(count: usize) -> Result<String> {
```

Comment:
there are probably other places where that gets handy as well :)

Reply:
Yes, we have lots of places where `iter().map(|_| "?").collect::<Vec<&str>>().join(",")` is done (just grep for it to find them). The time has come to unify them! (in this or a later PR)